src/OrleansRuntime/OrleansRuntime.csproj

```diff
@@ -186,7 +186,6 @@
     <Compile Include="Streams\QueueBalancer\StaticClusterDeploymentConfiguration.cs" />
     <Compile Include="Streams\QueueBalancer\ConsistentRingQueueBalancer.cs" />
     <Compile Include="Streams\QueueBalancer\DeploymentBasedQueueBalancer.cs" />
-    <Compile Include="Streams\QueueBalancer\IDeploymentConfiguration.cs" />
```

Comment:
NM..  Other queue balancer interfaces are already in PersistentStreams.  Just moving the remaining interface.
We can/should move queue balancer outside of streams to be more widely used, but we can do that if/when needed.  In any case, outside the scope of this change.